package.json

```diff
@@ -40,16 +40,16 @@
     "node": ">=8.0.0"
   },
   "devDependencies": {
-    "@babel/cli": "^7.8.4",
-    "@babel/core": "^7.9.6",
-    "@babel/preset-env": "^7.9.6",
-    "@metarhia/doc": "^0.6.0",
+    "@babel/cli": "^7.10.1",
+    "@babel/core": "^7.10.2",
+    "@babel/preset-env": "^7.10.2",
+    "@metarhia/doc": "^0.6.1",
     "eslint": "^7.1.0",
-    "eslint-config-metarhia": "^7.0.0",
+    "eslint-config-metarhia": "^7.0.1",
     "eslint-config-prettier": "^6.11.0",
     "eslint-plugin-import": "^2.20.2",
     "eslint-plugin-prettier": "^3.1.3",
     "metatests": "^0.7.1",
-    "prettier": "1.18.x"
+    "prettier": "^1.18.0"
```

Comment:
Prettier is known to break (drastically change the formatting of code) in minor releases that's what we put it at a patch-only level.

Reply:
It looks like that's exactly what happened due to the updated prettier version, hence the broken CI.

Reply:
I'dd add tests to prettier ignore because `tests/` readability is more important and will fix `lib/` @lundibundi

Reply:
Also I didn't update to the last prettier because it fixes `x => x` to `(x) => x` :smile:

Reply:
I'd prefer to have such [ignore] rules only for declarative tests to still get proper formatting of imperative tests, we could i.e. just add prettier ignore statements for declarative tests or put them in a separate folder.

Reply:
> Also I didn't update to the last prettier because it fixes x => x to (x) => x :)

This is fixable, they just changed the default.
https://prettier.io/docs/en/options.html#arrow-function-parentheses